src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -325,6 +327,10 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->validate()
+                            ->ifTrue(function ($v) { return isset($v['supports']) && isset($v['support_strategy']); })
```

Comment:
``supports`` will always be set as it is a prototyped array node and so has a default. You need to check whether it is the empty array instead.